src/main/java/org/reactome/release/qa/check/NewRegulationChecker.java

```diff
@@ -0,0 +1,84 @@
+
+    @Override
+    public String getDisplayName()
+    {
+        // Output report file name.
+        return "ReactionlikeEvent_Regulations_Not_Reviewed";
+    }
+
+    @Override
+    public void setOtherDBAdaptor(MySQLAdaptor adaptor)
+    {
+        this.priorAdaptor = adaptor;
+    }
+
+    @Override
+    public QAReport executeQACheck() throws Exception
+    {
+        QAReport report = new QAReport();
+        report.setColumnHeaders("DBID","DisplayName", "SchemaClass", "Issue", "MostRecentAuthor");
+        // Find all ReactionlikeEvents that have a filled 'regulatedBy' attribute in current slice.
+        Set<GKInstance> currentRlEsWithRegulations = getRlEsWithRegulationInstances(dba);
+
+        for (GKInstance currentRlE : currentRlEsWithRegulations) {
+            GKInstance previousRlE = priorAdaptor.fetchInstance(currentRlE.getDBID());
+            // QA check
+            if (newRegulationWithoutReviewed(currentRlE, previousRlE)){
+                report.addLine(currentRlE.getDBID().toString(), currentRlE.getDisplayName(), currentRlE.getSchemClass().getName(), "ReactionlikeEvent with new regulatedBy instance but has not yet been reviewed", QACheckerHelper.getLastModificationAuthor(currentRlE));
+            }
+        }
+        return report;
+    }
+
+    /**
+     * This method checks to see if a new regulation instance has been added to the ReactionlikeEvent by comparing between slices.
+     * If it does, it then compares the number of reviewed instances between slices. If the number of regulation instances is greater AND
+     * the number of reviewed instances is the same, it is flagged to indicate that this ReactionlikeEvent still needs to reviewed before
+     * it can be released.
+     * @param currentRlE GKInstance -- ReactionlikeEvent from current slice.
+     * @param previousRlE GKInstance -- Same ReactionlikeEvent, but from previous slice.
+     * @return boolean -- True if a new, unreviewed regulation instance has been added.
+     * @throws Exception -- DBA exceptions.
+     */
+    private boolean newRegulationWithoutReviewed(GKInstance currentRlE, GKInstance previousRlE) throws Exception {
+        // If previousRlE does not exist, we ignore it since that does not pertain to this particular QA check.
+        if (previousRlE != null) {
+            int currentRlERegulationCount = currentRlE.getAttributeValuesList(ReactomeJavaConstants.regulatedBy).size();
+            int currentRlEReviewedCount = currentRlE.getAttributeValuesList(ReactomeJavaConstants.reviewed).size();
+            int previousRlERegulationCount = previousRlE.getAttributeValuesList(ReactomeJavaConstants.regulatedBy).size();
+            int previousRlEReviewedCount = previousRlE.getAttributeValuesList(ReactomeJavaConstants.reviewed).size();
+
+            // The actual QA check
+            if (currentRlERegulationCount > previousRlERegulationCount && currentRlEReviewedCount == previousRlEReviewedCount) {
```

Comment:
Simply based on the numbers of regulatedBy instances may not be right: e.g. the curator may delete two regulateBy instances and then add one or just replace one and then add another one. You may have to check the actual content here. Please see this as an example: https://github.com/reactome/CuratorTool/blob/b1599c347d8761e02316d40638758344585b6b29/src/org/gk/model/InstanceUtilities.java#L569.

Reply:
@guanmingwu I tried to use the `compareAttValues` method (https://github.com/reactome/CuratorTool/blob/b1599c347d8761e02316d40638758344585b6b29/src/org/gk/model/InstanceUtilities.java#L626) in InstanceUtilities but it isn't working as expected.

When I try `compareAttValues(currentRlE, previousRlE, regulatedBySchemaAttribute)` I get exception 'Cannot cast GKInstance to List'. This exception makes sense based on the method. But what do I provide to the method to compare the 'regulatedBy' values of two ReactionlikeEvent instances? 

I've tried submitting both RlE's "regulatedBy" Lists to the method, and I get an unexpected exception `Invalid attribute 'regulatedBy' for class 'Reaction'`, which doesn't seem right.

I've tried submitting both RlEs as a single-entry List. This doesn't throw an Exception but it always returns true, which is wrong.

Is this method meant to be used this way? Anyways, this is a relatively simple task so I'm just going to compare the values of 'regulatedBy' within this code.